torch/csrc/api/include/torch/fft.h

```diff
@@ -5,25 +5,53 @@
 namespace torch {
 namespace fft {
 
-#ifndef DOXYGEN_SHOULD_SKIP_THIS
-namespace detail {
-
-inline Tensor fft(const Tensor& self) {
-  return torch::fft_fft(self);
-}
-
-} // namespace detail
-#endif /* DOXYGEN_SHOULD_SKIP_THIS */
-
 /// See the documentation of torch.fft.fft.
 ///
 /// Example:
 /// ```
 /// auto t = torch::randn(128, dtype=kComplexDouble);
 /// torch::fft::fft(t);
 /// ```
-inline Tensor fft(const Tensor& self) {
-  return detail::fft(self);
+inline Tensor fft(const Tensor& self,
+                  c10::optional<int64_t> n=c10::nullopt,
+                  int64_t axis=-1,
+                  c10::optional<std::string> norm=c10::nullopt) {
+  return torch::fft_fft(self, n, axis, norm);
+}
+
+inline Tensor ifft(const Tensor& self,
```

Comment:
Nit: simple doxygen comments for these, too. The torch:fft:fft documentation looks like this:

https://pytorch.org/cppdocs/api/function_namespacetorch_1_1fft_1ab483f290d94407a181ebb9793733d278.html?highlight=fft

Unfortunately it creates a link out of torch.fft.fft that doesn't work, though. @glaringlee maybe there's a better way to reference the Python documentation?

Reply:
@mruberry doxygen generates its documents within c++ domain. It can not ref python docs. We can manually add it in, but we don't have any way to add it automatically, they are total different domains. 
See this link:
https://pytorch.org/cppdocs/api/classtorch_1_1nn_1_1_multihead_attention_impl.html#classtorch_1_1nn_1_1_multihead_attention_impl
We put a doxygen style comments in MultiheadAttention impl, so it shows where is the python doc.

Reply:
Got it. Thanks @glaringlee!

Reply:
Added simple doxy-comments with a direct link to the python docs.